spec/controllers/chargeback_controller_spec.rb

```diff
@@ -110,4 +111,25 @@
       expect(flash_array.first[:message]).to include("rate is assigned and cannot be deleted")
     end
   end
+  context "#get_cis_all" do
```

Comment:
maybe use `describe` instead of `context` and add here new line, we are using `describe` for methods and `context` for cases with certain conditions, etc. ? is it right @chrisarcand ?